code/game/gamemodes/wizard/wizard.dm

```diff
@@ -158,22 +158,30 @@
 	wizard_mob.gene_stability += DEFAULT_GENE_STABILITY //magic
 	return 1
 
+// Checks if the game should end due to all wizards and apprentices being dead, or MMI'd/Borged
 /datum/game_mode/wizard/check_finished()
 	var/wizards_alive = 0
 	var/traitors_alive = 0
+
+	// Wizards
 	for(var/datum/mind/wizard in wizards)
 		if(!istype(wizard.current,/mob/living/carbon))

```

Comment:
shouldn't this already catch the wizards in an MMI? since it's not a `/mob/living/carbon` but an `/obj/item/mmi`